compose/services/docker-compose.mailcatchers.yml

```
@@ -0,0 +1,48 @@
+
+    networks:
+      - internal
+      - web
+
+  slurper:
+    image: cycloid/mailslurper:1.14.1
+    container_name: mail-slurper
+    hostname: mail-slurper
+    networks:
+      - internal
+      - web
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)